logIn/views.py

```diff
@@ -6,18 +6,21 @@
 
 
 def login_user(request, user_type):
+    valid_error = False
     login_error = False
     if request.method == "POST":
         form = LoginForm(request.POST)
         if form.is_valid():
             username = form.cleaned_data["username"]
             password = form.cleaned_data["password"]
             if user_type == UserType.STUDENT:
-                user = Student.objects.filter(username=username, password=password)
-                if user:
-                    return HttpResponse("Yay! We do remember you... (Student)")
-                else:
+                user = Student.objects.get(username=username)

```

Comment:
`get` will throw an exception if object is not found, where as `filter` returns an empty queryset. Either use try-catch or use filter()

Reply:
updated to reflect this change